### PR TITLE
Fix #4063 - fix Ubuntu 18.04 setup steps & PS vers

### LINKED ISSUE #4063: Update install instructions for Ubuntu 18.04
See https://github.com/PowerShell/PowerShell/issues/8008 for details

Issue Details
-----------------

Version(s) of document impacted
------------------------------
- [X] Impacts 6.next document
- [X] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------
- [ ] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[d92424b](https://github.com/sdwheeler/PowerShell-Docs/pull/4131/commits/d92424b347c32363b2ca9693a80bc0e2d6380d6b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4063/reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/install/installing-powershell-core-on-linux?view=powershell-3.0&branch=pr-en-us-4131) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/install/installing-powershell-core-on-linux?view=powershell-4.0&branch=pr-en-us-4131) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/install/installing-powershell-core-on-linux?view=powershell-5.0&branch=pr-en-us-4131) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/install/installing-powershell-core-on-linux?view=powershell-5.1&branch=pr-en-us-4131) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/install/installing-powershell-core-on-linux?view=powershell-6&branch=pr-en-us-4131) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C8%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904082027041629-4131%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=239PeZJMQSqmV3%2BY21TYlaA%2BoS2zZbdFeZX5f%2F7kAeU%3D&st=2019-04-08T20%3A26%3A56Z&se=2019-05-09T20%3A31%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 1]
```diff
@@ -415,19 +393,19 @@ sudo yum remove powershell
 zypper update && zypper --non-interactive install curl tar libicu52_1
 
 # Download the powershell '.tar.gz' archive
-curl -L https://github.com/PowerShell/PowerShell/releases/download/v6.1.0/powershell-6.1.0-linux-x64.tar.gz -o /tmp/powershell.tar.gz
+curl -L https://github.com/PowerShell/PowerShell/releases/download/v6.2.0/powershell-6.2.0-linux-x64.tar.gzz -o /tmp/powershell.tar.gz
```
Comment (DCtheGeek) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md]: Looks like an extra z in the link here.

Comment (sdwheeler) [reply reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md]: fixed

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[0928b89](https://github.com/sdwheeler/PowerShell-Docs/pull/4131/commits/0928b8902bbd044e85d9c81f9dae6c289b41272f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4063/reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/install/installing-powershell-core-on-linux?view=powershell-3.0&branch=pr-en-us-4131) (powershell-3.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/install/installing-powershell-core-on-linux?view=powershell-4.0&branch=pr-en-us-4131) (powershell-4.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/install/installing-powershell-core-on-linux?view=powershell-5.0&branch=pr-en-us-4131) (powershell-5.0)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/install/installing-powershell-core-on-linux?view=powershell-5.1&branch=pr-en-us-4131) (powershell-5.1)<br/> [View](https://review.docs.microsoft.com/en-us/PowerShell/scripting/install/installing-powershell-core-on-linux?view=powershell-6&branch=pr-en-us-4131) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C8%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904082038175708-4131%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=IN1wITeRe0Cz2AyUllweS%2ByJY3yLFQasKb0IG%2FF%2BzXQ%3D&st=2019-04-08T20%3A38%3A09Z&se=2019-05-09T20%3A43%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**